src/main/frontend/handler/events.cljs

```diff
@@ -204,20 +205,27 @@
   (p/let [content (when content (encrypt/decrypt content))]
     (state/set-modal! #(git-component/file-specific-version path hash content))))
 
-(defmethod handle :after-db-restore [[_ repos]]
-  (mapv (fn [{url :url}]
-          ;; compare :ast/version
-          (let [db (conn/get-conn url)
-                ast-version (:v (first (d/datoms db :aevt :ast/version)))]
-            (when (and (not= config/local-repo url)
-                       (or (nil? ast-version)
-                           (. semver lt ast-version db-schema/ast-version)))
-              (notification/show!
-               [:p.content
-                (util/format "DB-schema updated, Please re-index repo [%s]" url)]
-               :warning
-               false))))
-        repos))
+;; defunced in https://github.com/logseq/logseq/commit/097d47a860dd234715d670a78f61dddfe7ca698f
+;; (defmethod handle :after-db-restore [[_ repos]]
+;;   (mapv (fn [{url :url}]
+;;           ;; compare :ast/version
+;;           (let [db (conn/get-conn url)
+;;                 ast-version (:v (first (d/datoms db :aevt :ast/version)))]
+;;             (when (and (not= config/local-repo url)
+;;                        (or (nil? ast-version)
+;;                            (. semver lt ast-version db-schema/ast-version)))
+;;               (notification/show!
+;;                [:p.content
+;;                 (util/format "DB-schema updated, Please re-index repo [%s]" url)]
+;;                :warning
+;;                false))))
+;;         repos))
+
+;; TODO: when "only restore the current graph instead of all the graphs" is done,
+;; remove invoke of :graph/ready in graph/switch and restore-and-setup!
+(defmethod handle :graph/ready [[_ repo]]
+  (js/console.log "graph ready")
```

Comment:
This looks like leftover debugging. If it's for logging, can we use `lambdaisland.glogi` instead?

Reply:
The removal is done